lib/Config.ts

```diff
@@ -27,6 +27,8 @@ class Config {
   public dbpath: string;
   /** Whether matching will be disabled */
   public nomatching: boolean;
+  /** Whether to perform sanity checks */
+  public sanitychecks: boolean;
```

Comment:
"sanity checks" is a little ambiguous but maybe I'll use this same option to disable the sanity swaps in which case it's a good thing it's not too specific. Maybe just add a some detail to the comment like "Whether to check that orders can possibly be swapped before adding them to the order book, can be disabled for testing & debugging purposes."

Another consideration is that if we ever pass a value for this option from the command line (which I think could happen if I reuse it for disabling #822) then we might want to call it `nosanitychecks` so that it can be set with `--nosanitychecks` from the command line.

Reply:
Good idea. I've renamed the config option to `nosanitychecks` and updated the comment.